Xamarin.Forms.Core/CameraView.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.ComponentModel;
+using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms
+{
+	[RenderWith(typeof(_CameraViewRenderer))]
+	public class CameraView: View, IElementConfiguration<CameraView>
+	{
+		public event EventHandler<MediaCapturedEventArgs> MediaCaptured;
+
+		public event EventHandler<string> MediaCaptureFailed;
+
+		public event EventHandler<bool> OnAvailable;
```

Comment:
@paymicro how about event OnMediaAvailable ?
I'm not sure, but I remember that we have that event on your old code. We can register handler for this event to process frame eg (face detection, text detection...) once is ready from Camera control